apps/web-connect/tsconfig.json

```diff
@@ -8,6 +8,7 @@
       "@sentry/*": ["../../packages/*/src"]
     }
   },
+  "include": ["src/assets/pagecontent/**/*.json", "src/**/*"],
```

Comment:
i dont think you need to include the .json files in the TS compiler. `"resolveJsonModule": true ` should handle that case?

Reply:
I actually got a build error trying to build without it and tried to go with the quickest fix.
`"resolveJsonModule": true` is still required I think.